images/fabric-ca-fvt/payload/start.sh

```diff
@@ -8,21 +8,27 @@
 POSTGRES_PORT=5432
 MYSQL_PORT=3306
 LDAP_PORT=389
-PORTS=($POSTGRES_PORT $MYSQL_PORT $LDAP_PORT)
+PORTS=($MYSQL_PORT $LDAP_PORT)
```

Comment:
I'd expect all of these to be set with defaults:

 PGDATABASE, PGHOST, PGPORT and/or PGUSER

```
PGDATABASE=${PGDATABASE:-postgres}
PGHOST=${PGHOST:-postgres}
PGPORT=${PGPORT:-5432}
PGUSER=${PGUSER:-postgres}

These are from the psql man page. https://www.postgresql.org/docs/9.3/libpq-envars.html